test/unit/tabs.test.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import Tabs, { TabLink } from '@auth0/cosmos/molecules/tabs/tabs'
+
+function tabsFactory() {
+  const content = {
+    first: <div className="content-1" />,
+    second: <div className="content-2" />,
+    third: <div className="content-3" />
+  }
+
+  const generator = (index = 0, onSelect = () => {}) =>
+    shallow(
+      <Tabs selected={index} onSelect={onSelect}>
+        <Tabs.Tab title="Title 1">{content.first}</Tabs.Tab>
+        <Tabs.Tab title="Title 2">{content.second}</Tabs.Tab>
+        <Tabs.Tab title="Title 3">{content.third}</Tabs.Tab>
+      </Tabs>
+    )
+
+  return { generator, content }
+}
+
+describe('Tabs tests', () => {
+  it('renders only selected tab content', () => {
+    const { generator, content } = tabsFactory()
+
+    // Create three <Tabs> instances with different tabs selected
+    const [first, second, third] = [generator(0), generator(1), generator(2)]
+
+    expect(first.contains(content.first)).toBe(true)
```

Comment:
Do you think it would easier to maintain snapshot tests for rendering stuff?

Reply:
Yes! Changed it :)